nflow-rest-api-common/src/main/java/io/nflow/rest/v1/ResourceBase.java

```diff
@@ -149,24 +149,15 @@ public boolean updateWorkflowInstance(final long id,
       String stateVariableKey, String stateVariableValue, String include, Long maxResults, Long maxActions, boolean queryArchive,
       WorkflowInstanceService workflowInstances, ListWorkflowInstanceConverter listWorkflowConverter) {
     Set<String> includeStrings = parseIncludeStrings(include).collect(toSet());
-    QueryWorkflowInstances q = new QueryWorkflowInstances.Builder()
-        .addIds(ids.toArray(new Long[ids.size()]))
-        .addTypes(types.toArray(new String[types.size()]))
-        .setParentWorkflowId(parentWorkflowId)
-        .setParentActionId(parentActionId)
+    QueryWorkflowInstances q = new QueryWorkflowInstances.Builder().addIds(ids.toArray(new Long[ids.size()]))
+        .addTypes(types.toArray(new String[types.size()])).setParentWorkflowId(parentWorkflowId).setParentActionId(parentActionId)
```

Comment:
Should we add `//` after each method to keep them on separate lines?

Reply:
nope, I just need to fix the formatter settings